app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -56,23 +64,33 @@
     "Eritrea",
     "Estonia",
     "Ethiopia",
+    "Falkland Islands",
     "Fiji",
     "Finland",
     "France",
+    "French Guiana",
+    "French Polynesia",
```

Comment:
https://en.wikipedia.org/wiki/ISO_3166-1 also lists "French Southern Territories", but I'm not sure whether we should be including these here or not as they don't have a permanent population.